packages/playwright/src/reporters/base.ts

```diff
@@ -547,3 +547,26 @@ function fitToWidth(line: string, width: number, prefix?: string): string {
 function belongsToNodeModules(file: string) {
   return file.includes(`${path.sep}node_modules${path.sep}`);
 }
+
+function resolveFromEnv(name: string): string | undefined {
+  const value = process.env[name];
+  if (value)
+    return path.resolve(process.cwd(), value);
+  return undefined;
+}
+
+export function resolveOutputFile(reporterName: string, options: { configDir: string, outputFile?: string }): string|undefined {
+  const name = reporterName.toUpperCase();
+  let outputFile = resolveFromEnv(`PLAYWRIGHT_${name}_OUTPUT_FILE`);
+  if (!outputFile && options.outputFile)
+    outputFile = path.resolve(options.configDir, options.outputFile);
+  // Explicit `outputFile` overrides `outputDir` and `fileName` options.
+  if (!outputFile) {
+    const outputName = process.env[`PLAYWRIGHT_${name}_OUTPUT_NAME`];
+    if (outputName) {
```

Comment:
1. We should resolve output name relative to the cwd for backwards compatibility, w/o documenting it.
2. Just passing PLAYWRIGHT_BLOB_OUTPUT_DIR should produce reports with default names in that folder

Reply:
> We should resolve output name relative to the cwd for backwards compatibility, w/o documenting it.

We already do in line 567.

> Just passing PLAYWRIGHT_BLOB_OUTPUT_DIR should produce reports with default names in that folder

This change does not affect blob reports, did you mean something else?